internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -13,11 +14,20 @@ import (
 	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
 )
 
+type AmsBillingModel interface {
```

Comment:
I find this name confusing.

AmsBillingModel is an AMS specific attribute. Which is independent of the AMS product and resources. At the same time this interface has GetAMSBillingModels, adding to the confusion.
I think we should think of an alternative name for this if possible

Reply:
Going to remove it in this PR, and I will rethink the name.